### PR TITLE
Updated: packages

### PR DESCRIPTION
**Pull request by GoliathLabs**
Updated a few packages with their respective new upstream URLs. One thing though, websploit now requires the python-requests package which is satisfied on my Arch Linux system but I couldn't find it within blackarch. Should I add it to the packages?

One more question, where did the Wireshark package go? It isn't available in this repository anymore :/

Cheers,
Felix Golatofski

### CODE REVIEW

File: packages/websploit/PKGBUILD [hunk 1]
```diff
@@ -2,29 +2,27 @@
...
 
-  tar xzf wsf-`echo $pkgver | tr -d '.'`.tar.gz
+  cd "$srcdir/$pkgname-$pkgver/$pkgname"
 
-  cd "$srcdir/$pkgname/$pkgname"
-
-  install -Dm 644 -t "$pkgdir/usr/share/doc/$pkgname/" README.txt
-  install -Dm 644 COPYING.GPL "$pkgdir/usr/share/licenses/$pkgname/LICENSE"
+  install -Dm 644 -t "$pkgdir/usr/share/doc/$pkgname/" ../README.md
+  install -Dm 644 ../LICENSE.txt "$pkgdir/usr/share/licenses/$pkgname/LICENSE"
```
Comment (Edu4rdSHL) [inline packages/websploit/PKGBUILD:25]: Please use:

```
  install -Dm 644 -t "$pkgdir/usr/share/doc/$pkgname/" README.md
  install -Dm 644 LICENSE.txt "$pkgdir/usr/share/licenses/$pkgname/LICENSE"

  cp -a --no-preserve=ownership $pkgname/* "$pkgdir/usr/share/websploit"
```
and delete the `cd "$srcdir/$pkgname-$pkgver/$pkgname"` line.

**Comment by Edu4rdSHL** [comment]
@GoliathLabs thank you for the contribution. Question do you are testing the packages before uploading the PKGBUILDs? See https://github.com/Edu4rdSHL/blackarch-devtools for a easy and accurate way to test packages locally. It's the same as clean chroot environment but automating all the stuff including executing packages in the clean chroot environment.

Regards,
Eduard

**Comment by GoliathLabs** [comment]
@Edu4rdSHL Thank you for the proposal. I've added the binary to my local build system. Might you give me an example for the websploit package so I can test further packages properly before submitting them?

Cheers,
Felix Golatofski

**Comment by Edu4rdSHL** [comment]
@GoliathLabs regarding to the `python-requests` package, it's available in official ArchLinux repos, so just add this as dependency.

For ba-devtools then just use:

```
ba-dev -s # To setup the environment.
```
Then go to the folder where your PKGBUILD is and execute `ba-dev -b # To build the package. `
And if this build successfully then test the package with: `ba-dev -e "command_to_execute" -p path/to/package.pkg.tar.zst`

**Comment by Edu4rdSHL** [comment]
Thanks, did you finally set up blackarch-devtools?

**Pull request merged**